source/dotnet/wholesale-api/CalculationResults/CalculationResults.IntegrationTests/Infrastructure/RequestCalculationResult/RequestCalculationResultQueriesTests.cs

```diff
@@ -385,7 +385,7 @@ public RequestCalculationResultQueriesTests(DatabricksSqlStatementApiFixture fix
 
     [Theory]
     [InlineAutoMoqData]
-    public async Task GetAsyncRequestFromGridOperatorTotalProductionThirdCorrectionSettlement_ReturnsResult(
+    public async Task GetAsync_RequestFromGridOperatorTotalProductionThirdCorrectionSettlement_ReturnsResult(
```

Comment:
We have decided that our test names must follow a pattern like `<MemberName>_When<Something>_<IsOrDoes>` so I have added "When": 
```suggestion
    public async Task GetAsync_WhenRequestFromGridOperatorTotalProductionThirdCorrectionSettlement_ReturnsResult(
```